source/developer-guides/configuration.rst

```diff
@@ -24,53 +18,61 @@ Internal configuration data
 External configuration data
   Platform specific data which is configurable by tools. It can be protected with user provided key.
 
-.. _Configuration Files and Configuration Flow:
+|SPN| configuration infrastructure includes the following and is designed to support multiple boards with one firmware image. 
+
+* Configuration declarations in YAML files
+* Configuration deltas in DLT files
+* Configuration tools
 
-One firmware image can support configuration requirements of multiple platforms. To support this, Slim Bootloader solution creates DSC file and DLT file to handle board level differences.
 
-.. image:: /images/Config1.jpg
-         :width: 600
-         :alt: Install Ubuntu 1 of 5
+.. _Configuration Files and Configuration Flow:
 
 Configuration Flow
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
-.. image:: /images/Config2.jpg
-         :width: 600
-         :alt: Install Ubuntu 1 of 5
+.. image:: /images/ConfigFlow.png
+
+Configuration Editor Tool
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+|SPN| supports a Configuration Editor Tool (ConfigEditor.py) to configure firmware settings with graphics UI. This tool is included in |SPN| source package at /SblOpen/BootloaderCorePkg/Tools.
+
+.. image:: /images/CfgEditOpen.png
+
 
-_DSC File
+YAML Files
 ^^^^^^^^^^^^
-* DSC file is the heart of all platform settings including memory, silicon, GPIO, OS boot policy, security settings etc.
-* DSC file, in general is located in project specific board folder. 
-* For example, you can find DSC file for APL platform under SblOpen\Platform\ApollolakeBoardPkg\CfgData folder.
+* All platform configuration settings including memory, silicon, GPIO, OS boot policy, security settings etc are declared in a custom format and uses YAML Syntax.
+* YAML configuration files, in general are located in project specific board folder. 
+* For example, you can find the configuration files for Apollo Lake platform under SblOpen\Platform\ApollolakeBoardPkg\CfgData folder.

```

Comment:
For SBL path, maybe better to make it consistent.  Use  Platform\ApollolakeBoardPkg\CfgData instead ?